app/javascript/blacklight/modal.js

```diff
@@ -101,12 +101,16 @@ Blacklight.modal.modalCloseSelector   = '[data-blacklight-modal~=close]';
 // Called on fatal failure of ajax load, function returns content
 // to show to user in modal.  Right now called only for extreme
 // network errors.
-Blacklight.modal.onFailure = function(data) {
-  var contents =  '<div class="modal-header">' +
-            '<div class="modal-title">Network Error</div>' +
-            '<button type="button" class="blacklight-modal-close close" data-dismiss="modal" aria-label="Close">' +
-            '  <span aria-hidden="true">&times;</span>' +
-            '</button>';
+Blacklight.modal.onFailure = function(jqXHR, textStatus, errorThrown) {
+  console.error('Server error:', this.url, jqXHR.status, errorThrown)
+  var contents =  `<div class="modal-header">
```

Comment:
Just dropping in the support matrix for template literals. This change would not be parsable by IE11.

https://caniuse.com/?search=template%20literals

Reply:
I thought we heard agreement on https://github.com/projectblacklight/blacklight/issues/2332 at blacklight conf.

Reply:
Maybe we did? Honestly I don't remember.